src/Components/PromiseList/index.js

```diff
@@ -10,11 +10,14 @@ export type Props = {
 
 const PromiseList = (props: Props) => {
   const { promises } = props;
+  if (!promises) {
```

Comment:
Is this check needed? Wont promises just be an empty array and thus the map function wont create any markup?